src/riscv_opcodes/go_utils.py

```diff
@@ -9,12 +8,19 @@
 logging.basicConfig(level=logging.INFO, format="%(levelname)s:: %(message)s")
 
 
-def make_go(instr_dict: InstrDict):
-
-    args = " ".join(sys.argv)
-    prelude = f"""// Code generated by {args}; DO NOT EDIT."""
-
-    prelude += """
+def make_go(instr_dict: InstrDict, extensions: list[str]):
+    comment = """// Code generated with following arguments; DO NOT EDIT.
+/*
+make inst.go"""
+    if extensions:
+        comment += " EXTENSIONS='\\\n"
```

Comment:
As these are going to be inside comment `/*...*/`, I believe that the backslashes have no effect on Go compilation.